notes.md

```diff
@@ -129,114 +117,128 @@ As an example a file named `intro.mkv` could have a `SegmentUID` of `0xb16a58609
 
-# Track Operation
+This is the type of problem that `TrackTimecodeScale` was designed to fix. Using it, the video can be played back at a speed that will synch with either the NTSC or the PAL audio stream, depending on which is being used for playback.
+To continue the above example:
 
-TrackOperation allows combining multiple tracks to make a virtual one. It uses 2 separate system to combine tracks. One to create a 3D "composition" (left/right/background planes) and one to simplify join 2 tracks together to make a single track.
+    Track 1: Video
+    Track 2: NTSC Audio
+    Track 3: PAL Audio
 
-A track created with TrackOperation is a proper track with a UID and all its flags. However the codec ID is meaningless because each "sub" track needs to be decoded by its own decoder before the "operation" is applied. The Cues corresponding to such a virtual track SHOULD be the sum of the Cues elements for each of the tracks it's composed of (when the Cues are defined per track).
+Because the NTSC track is at the original speed, it will used as the default value of 1.0 for its `TrackTimecodeScale`. The video will also be aligned to the NTSC track with the default value of 1.0.
 
-In the case of TrackJoinBlocks, the Blocks (from BlockGroup and SimpleBlock) of all the tracks SHOULD be used as if they were defined for this new virtual Track. When 2 Blocks have overlapping start or end timecodes, it's up to the underlying system to either drop some of these frames or render them the way they overlap. In the end this situation SHOULD be avoided when creating such tracks as you can never be sure of the end result on different platforms.
+The `TrackTimecodeScale` value to use for the PAL track would be calculated by determining how much faster the PAL track is than the NTSC track. In this case, because we know the video for the NTSC audio is being played back at 24fps and the video for the PAL audio is being played back at 25fps, the calculation would be:
+
+    (25 / 24) = ~ 1.04166666666666666667
```

Comment:
Why parentheses?

Reply:
I can't answer that; that's how it was in the original source.

Reply:
And I would use `≈` instead of `= ~`

Reply:
:+1: to proper Unicode symbols.

Reply:
Updated. Still going to wait until other nearby PRs are taken care of before resolving conflicts below.